api/javatests/org/openyolo/api/CredentialClientTest.java

```diff
@@ -146,17 +132,6 @@ public void getInstance_returnsNonNull() throws Exception {
 
     @SuppressWarnings("WrongConstant")
     @Test
-    public void testGetHintIntent_noProviders() throws Exception {
```

Comment:
Maybe keep these 'noProviders' tests and just assert the correct FinishWithResultActivity intent is returned?

Reply:
Done.